### PR TITLE
init rel download shell script

### PR DESCRIPTION
**Pull request by ransomw1c**
this is the "manual curl command" option (via github releases) to push updates to users and address #142

### CODE REVIEW

File: hack/github_release_run.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+
+# todo: replace release id with /latest after non-pre-release exists
+# REL_URL=https://api.github.com/repos/oneconcern/datamon/releases/latest
+REL_URL=https://api.github.com/repos/oneconcern/datamon/releases/16541641
```
Comment (kerneltime) [inline hack/github_release_run.sh]: We will cut a release soon that works with latest, you can flip this to latest. Will merge it in and cut the release in this week.

**Comment by kerneltime** [comment]
The script can be in deploy and called datamon?

**Comment by kerneltime** [comment]
We can also include the MD5 sum as part of releases and include them in the script.

**Comment by ransomw1c** [comment]
yes, some checksum ought to be added with regard to security.

File: deploy/datamon.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#! /bin/sh
+
+# set these to non-zero to enable
```
Comment (kerneltime) [inline deploy/datamon.sh]: Please add some documentation for that this script is expected to do.

**Pull request merged**